src/domain/grid_domain/grid.jl

```diff
@@ -0,0 +1,211 @@
+using Polyhedra
```

Comment:
We should favor `import` over `using` when it is inside the code of a package. It greatly improves readability because the reader directly sees that the function belongs to the `Polyhedra` package

Reply:
I've clarified it in most places, later I'll make it even more explicit, but I'll do it when refactoring the other solvers.